e2e/config.js

```diff
@@ -54,6 +54,7 @@ module.exports = {
     changeCaseName: 'Change case name',
     submitCase: 'Submit application',
     deleteApplication: 'Delete an application',
+    draftCMO:'Draft CMO',
```

Comment:
space after colon, name should be complete as well imo

Reply:
done this change